README.md

```diff
@@ -117,6 +117,7 @@ dockerCompose {
     projectNamePrefix = 'my_prefix_' // allow to set custom prefix of docker-compose project name, the final project name has nested configuration name appended
     executable = '/path/to/docker-compose' // allow to set the path of the docker-compose executable (useful if not present in PATH)
```

Comment:
Please add a comment that this value is not used if `useDockerComposeV2` is set to `true`.